CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Prism is correctly handling the `csv` collection format argument property in OAS2 documents #577
 - Prism is correctly returning the response when the request has `*/*` as Accept header #578
 - Prism is correctly returning a single root node with the payload for XML data #578
+- Prism is correctly responding with the correct status code when security fails
```

Comment:
As mentioned in another PR this changelog item is not as descriptive as it could be,. 

```suggestion
- Requests lacking security credentials will return a 401 instead of the usual 422/400 validation error
```